client/css/style.css

```diff
@@ -678,9 +678,15 @@ button, .user {
 }
 #chat .part .from:before,
 #chat .quit .from:before {
-    font-family: FontAwesome;
+	font-family: FontAwesome;
 	content: "\f08b";
 	color: #FF4136;
+	display: inline-block;
+	-webkit-transform: rotate(180deg);
+	-moz-transform: rotate(180deg);
+	-ms-transform: rotate(180deg);
+	-o-transform: rotate(180deg);
+	transform: rotate(180deg);
```

Comment:
(Let's not do that, this is a fairly modern web app, it's reasonable to ask users to have recent browsers, and we can start ensuring compatibility when the project is more stable.)